app/journeys/probate.js

```diff
@@ -280,5 +283,125 @@ const stepList = {
     ReviewResponse: 'CitizensHub'
 };
 
+const previousStepList = {
+    Dashboard: 'StartApply',
+    StartEligibility: 'Dashboard',
+    DeathCertificate: 'StartEligibility',
+    StopPage: {
+        DeathCertificate: 'DeathCertificate',
+        DeathCertificateTranslation: 'DeathCertificateTranslation',
+        DeceasedDomicile: 'DeceasedDomicile',
+        ExceptedEstateValued: 'ExceptedEstateValued',
+        StartEligibility: 'StartEligibility'
+    },
+    DeathCertificateInEnglish: 'DeathCertificate',
+    DeathCertificateTranslation: 'DeathCertificateInEnglish',
+    DeceasedDomicile: 'DeathCertificateInEnglish',
+    ExceptedEstateDeceasedDod: 'DeceasedDomicile',
+    ExceptedEstateValued: 'ExceptedEstateDeceasedDod',
+    IhtCompleted: 'ExceptedEstateDeceasedDod',
+    WillLeft: 'ExceptedEstateValued',
+    WillOriginal: 'WillLeft',
+    ApplicantExecutor: 'WillOriginal',
+    MentalCapacity: 'ApplicantExecutor',
+    TaskList: 'Dashboard',
+    BilingualGOP: 'Dashboard',
+    DeceasedName: 'BilingualGOP',
+    DeceasedDob: 'DeceasedName',
+    DeceasedDod: 'DeceasedDob',
+    DeceasedAddress: 'DeceasedDod',
+    DiedEnglandOrWales: 'DeceasedAddress',
+    DeathCertificateInterim: 'DiedEnglandOrWales',
+    IhtEstateValued: 'DeathCertificateInterim',
+    IhtEstateForm: 'IhtEstateValued',
+    ProbateEstateValues: 'IhtEstateForm',
+    DeceasedAlias: 'ProbateEstateValues',
+    DeceasedOtherNames: 'DeceasedAlias',
+    DeceasedMarried: 'DeceasedOtherNames',
+    WillHasVisibleDamage: 'DeceasedMarried',
+    WillDamageReasonKnown: 'WillHasVisibleDamage',
+    WillDamageCulpritKnown: 'WillDamageReasonKnown',
+    WillDamageDate: 'WillDamageCulpritKnown',
+    WillCodicils: 'WillHasVisibleDamage',
+    CodicilsNumber: 'WillCodicils',
+    CodicilsHasVisibleDamage: 'CodicilsNumber',
+    CodicilsDamageReasonKnown: 'CodicilsHasVisibleDamage',
+    CodicilsDamageCulpritKnown: 'CodicilsDamageReasonKnown',
+    CodicilsDamageDate: 'CodicilsDamageCulpritKnown',
+    DeceasedWrittenWishes: {
+        CodicilsDamageDate: 'CodicilsDamageDate',
+        WillCodicils: 'WillCodicils'
+    },
+    ApplicantName: 'TaskList',
```

Comment:
multiExecPreviousStepList is the same here (L335) till the closing of this object.